packages/terra-paginator/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "terra-paginator",
+  "main": "lib/Paginator.js",
+  "version": "0.0.0",
+  "description": "Paginator to be used for data sets of known and unknown size. Provides first, last, previous, next, and paged functionality.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-paginator",
+    "Paginator",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-props-table": "^1.10.1"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
```

Comment:
Let's update these to the following:

```
    "react": "^16.2.0",
    "react-dom": "^16.2.0",
    "terra-base": "^3.7.0"
```

Reply:
https://github.com/cerner/terra-core/pull/1445/commits/96d34fbb060bb3ece55c6262cc1c68afa83e91af